shell/server/stats-server.js

```diff
@@ -182,6 +182,10 @@ function recordStats() {
     plans: planStats,
   };
   record.computeTime = Date.now() - now;
+  if (SandstormDb.getTotalCharges) {
+    // This only exists under Blackrock
+    record.totalCharges = SandstormDb.getTotalCharges();
```

Comment:
Perhaps this function should hang off of `BlackrockPayments` rather than monkey-patching `SandstormDb` then, if it only exists on Blackrock?